totalRP3/Modules/Toolbar/Toolbar.lua

```diff
@@ -50,9 +50,9 @@ function ToolbarMixin:Toggle()
 	self.forcedVisibility = nil;
 
 	if self:IsShown() then
-		TRP3_API.ui.misc.playUISound(SOUNDKIT.IG_MAINMENU_OPEN);
+		PlaySound(SOUNDKIT.IG_MAINMENU_OPEN);
 	else
-		TRP3_API.ui.misc.playUISound(SOUNDKIT.IG_MAINMENU_CLOSE);
+		PlaySound(SOUNDKIT.IG_MAINMENU_CLOSE);
```

Comment:
Should those not use one of the TRP3_InterfaceSounds?

Reply:
I'd thought about it but the toolbar is a bit of a special case, so I'm happy to just let it directly go out and figure out its soundkits for now.